_data/adobe-sid/adobe-sid.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SIDDHANT MANGLIK
+institution: JAYPEE INSTITUTE OF INFORMATION TECHNOLOGY 🚩 
```

Comment:
I would suggest that you remove the "flag-symbol" at the end to prevent any merge errors.